server/routes/incidentDetails/incidentDetailsValidation.ts

```diff
@@ -49,9 +51,68 @@ const errors: { [key: string]: FormError } = {
     href: '#incidentDate[time]',
     text: 'The incident time must be in the past',
   },
+  DISCOVERY_MISSING_RADIO: {
+    href: '#discoveryRadioSelected',
+    text: 'Select an incident discovery time opttion',
```

Comment:
Spelling mistake here - I missed this before, sorry

Reply:
Also, was this validation message in the AC's or did you write it? Usually if I have to create a new validation message I run the wording by Anne so I can be sure it's rights (we have to be very specific with our content).